tests/driver/nodejs/build.rs

```diff
@@ -29,6 +29,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         "cargo:rustc-env=SLINT_NODE_NATIVE_LIB={}",
         target_dir.join(nodejs_native_lib_name).display()
     );
+    println!("cargo:rustc-env=SLINT_ENABLE_EXPERIMENTAL_FEATURES=1",);
```

Comment:
Why do we need this if the test is anyway ignored.  And also you set it in the `.env`  in nodejs.rs